TBDataPreparation/202108_SPS/scripts/PhysicsEvent.h

```diff
@@ -52,30 +54,25 @@ PMTCalibration::PMTCalibration(const std::string& fname){
 }
 
 
-class EventOut{
-  public:
-	EventOut(){};
-	~EventOut(){};
-	uint32_t EventID;
-
-        float SPMT1, SPMT2, SPMT3, SPMT4, SPMT5, SPMT6, SPMT7, SPMT8;
-    	float CPMT1, CPMT2, CPMT3, CPMT4, CPMT5, CPMT6, CPMT7, CPMT8;
-        float SiPMPheC[160] = {0};
-        float SiPMPheS[160] = {0};
-        float totSiPMCene = 0.;
-        float totSiPMSene = 0.;
-        float SPMTenergy = 0.;
-        float CPMTenergy = 0.;
-        int PShower, MCounter, C1, C2;
-    	
-    	void CompSPMTene(){SPMTenergy = SPMT1+SPMT2+SPMT3+SPMT4+SPMT5+SPMT6+SPMT7+SPMT8;}
-        void CompCPMTene(){CPMTenergy = CPMT1+CPMT2+CPMT3+CPMT4+CPMT5+CPMT6+CPMT7+CPMT8;}
+struct EventOut{
+    EventOut(){};
+    ~EventOut(){};
+    uint32_t EventID;
+    float SPMT1, SPMT2, SPMT3, SPMT4, SPMT5, SPMT6, SPMT7, SPMT8;
+    float CPMT1, CPMT2, CPMT3, CPMT4, CPMT5, CPMT6, CPMT7, CPMT8;
+    float SiPMPheC[160] = {0};
+    float SiPMPheS[160] = {0};
+    float totSiPMCene = 0.;
+    float totSiPMSene = 0.;
+    float SPMTenergy = 0.;
+    float CPMTenergy = 0.;
+    int PShower, MCounter, C1, C2;
+    inline void CompSPMTene(){SPMTenergy = SPMT1+SPMT2+SPMT3+SPMT4+SPMT5+SPMT6+SPMT7+SPMT8;}
+    inline void CompCPMTene(){CPMTenergy = CPMT1+CPMT2+CPMT3+CPMT4+CPMT5+CPMT6+CPMT7+CPMT8;}
```

Comment:
About the CompSMPTene() and CompCPMTene(), if I understand correctly, there is a bug. You calculate the same quantity in the calibratePMT() method but leave the definition and implementation of the functions inside the struct EventOut. They should be removed. Could you please check.

Reply:
Yes I've moved theese in the calibration method and we don't need those anymore